### DISCUSSION TITLE
Neue Anlage, WR lässt sich nicht eintragen (2024.08.18) ==> "Unbekanntes Format"

### DISCUSSION THREAD

**Discussion by Matti07** [hoylabs/OpenDTU-OnBattery#1240] [Q&A] [answered]
Moin, bei einer nagelneuen Anlage bekomme ich den WR nicht eingetragen.
Versucht unter Linux, Win10 und Android sowie mit FF und Chrome. Cache löschen, Fenster neu laden, Rechner neu starten, ...
Ich kann eingeben was ich will, das Ergebnis ist immer identisch, siehe Hardcopy.
Beide Felder lassen sich ausfüllen, aber der WR lässt sich natürlich nicht hinzufügen.

![Open DTU unbekanntes format](https://github.com/user-attachments/assets/6c942a3d-1b3a-4004-bc79-2ea70db30608)

Was mache ich da falsch?

**Comment by spcqike** [comment]
Was ist es denn für ein Modell und Wie lautet denn die Seriennummer? „9“ ist natürlich Quatsch und falsch. 

Die ersten 4 oder 6 stellen reichen glaube ich.

**Comment by Matti07** [comment]
HMS-2000-4T, #1164931..
Ich habe u.a. die Info gefunden, dass man die ersten 4 Ziffern weglassen soll.
Ist aber wirklich egal was ich dort eintrage, 1 oder 9 oder ein Buchstabe zwischen A und F.
Auch die Nr in hex hat nichts geändert.

**Comment by spcqike** [comment, marked as answer]
Die Fehlermeldung kommt solange keine gültige Seriennummer eingetragen ist. 

Trage doch mal die komplette Seriennummer ein. Beginnend mit 1164. alle 12 stellen, wie sie auf dem wechselrichter stehen.

**Reply by Matti07** [reply]
DANKE!!! Das war`s. Schönen Sonntag noch.

**Reply by spcqike** [reply]
Super. 

Na dann, dir auch. Und noch ein paar sonnige Tage ;)
